developers/weaviate/search/image.md

```diff
@@ -63,6 +64,15 @@ If your query image is stored in a file, you can use the client library to searc
     />
   </TabItem>
 
+  <TabItem value="go" label="Go">
```

Comment:
This block should be added as last one (after `JS/TS Client v2`), now it's just after python client, which I think it's not a good place